package.json

```diff
@@ -1561,6 +1557,81 @@
 				}
 			]
 		},
+        "walkthroughs": [
+            {
+                "id": "openshiftWalkthrough",
+                "title": "Getting Started with OpenShift",
+                "description": "Start your application development on OpenShift or Kubernetes",
+                "steps": [
+                    {
+                        "id": "login",
```

Comment:
![image](https://user-images.githubusercontent.com/620330/171700521-56163318-d760-42b0-a806-c95be9a3d55f.png)

This step does not look right to me. Login is just a way to get access to cluster resources and it does not matter how cluster is provisioned or where it is located.

What I see is attempt to mix cluster login, provisioning and location in one step.

I suggest to bubble up one more step. 'Get access to OpenShift cluster' or 'Provision or use existing OpenShift cluster'
Where we can have button to open 'Add Cluster Editor'. Then reduce Login step to just a description and login button.

Reply:
![image](https://user-images.githubusercontent.com/93245779/173075447-2e922eb4-2850-454b-bef1-011ee7970810.png)
Changed the login title and description. Please refer the screenshot

Reply:
I think this covers the use case of differentiating the Login and provision workflow. The description also works for me. LGTM.

Reply:
Well, it is still two questions answered in one step.

'Connect using OpenShift Local' sounds awkward to me, I would say 'Connect using ssh' or 'Connect to OpenShift using SSH'. Both sounds good to me. Lets try 'Connect to OpenShift Using OpenShift Local' does that sound good? It is not for me.

IMHO first someone who is interested would ask where/how one can get OpenShift instance? Then ask how to work with it or see what is in it through the UI elements provided by extension they've just installed and that would require to log into the cluster.

Reply:
Here OpenShift local refers to renamed CodeReadyContainers. So connect using OpenShift local should open the CRC view we currently have.